precompile.sh

```diff
@@ -0,0 +1,10 @@
+handlebars public/components/Menu/Menu.hbs -f public/components/Menu/Menu.precompiled.js 
+handlebars public/components/MenuLink/MenuLink.hbs -f public/components/MenuLink/MenuLink.precompiled.js 
+handlebars public/components/Header/Header.hbs -f public/components/Header/Header.precompiled.js 
+handlebars public/components/Post/Post.hbs -f public/components/Post/Post.precompiled.js 
+handlebars public/components/Container/Container.hbs -f public/components/Container/Container.precompiled.js
+handlebars public/components/SignupForm/SignupForm.hbs -f public/components/SignupForm/SignupForm.precompiled.js
+handlebars public/components/LoginForm/LoginForm.hbs -f public/components/LoginForm/LoginForm.precompiled.js
+handlebars public/components/Input/Input.hbs -f public/components/Input/Input.precompiled.js
+handlebars public/components/FormButton/FormButton.hbs -f public/components/FormButton/FormButton.precompiled.js
+handlebars public/components/FormLink/FormLink.hbs -f public/components/FormLink/FormLink.precompiled.js
```

Comment:
Можно попробовать что-то такое, если хотите оставить баш скрипт

```suggestion
#!/bin/bash  

COMPONENTS_DIR="public/components"  

for hbs_file in "$COMPONENTS_DIR"/*/*.hbs; do  
    js_file="${hbs_file%.hbs}.precompiled.js"  

    handlebars "$hbs_file" -f "$js_file"  
done
```

И не забудьте добавить перевод строки, пожалуйста

У меня другой подход к этому делу, npm скрипт, который все хбски закидывает в один прекомпайлед, а его уже монтируем в html

```
"compile": "globstar -- npx handlebars source/**/*.hbs -f source/precompiled.js",
```